### DISCUSSION TITLE
Conditionally hide cftarget segment on `manifest.yml` file existence

### DISCUSSION THREAD

**Discussion by sonallux** [JanDeDobbeleer/oh-my-posh#3647] [Troubleshoot] [answered]
First, thank you for this project. I really like it 👍.

I often have to work with Cloud Foundry and therefore want to use the [cftarget segment](https://ohmyposh.dev/docs/segments/cftarget) in my prompt. But actually, the cf target information only really matters if I am in a folder with a `manifest.yml` file. So how do I hide the cftarget segment if the folder does not contain a `manifest.yml` file? I have seen that the [cf segment](https://ohmyposh.dev/docs/segments/cf) is only displayed when such a file exists. Unfortunately, the `display_mode` property does not exist for the cftarget segment.

Is there a way to conditionally hide segments based on file existence or should I open a feature request for that?

**Comment by JanDeDobbeleer** [comment, marked as answer]
https://github.com/JanDeDobbeleer/oh-my-posh/pull/3649

**Reply by sonallux** [reply]
Wow, that was quick 🚀 Thank you very much for adding this feature 👍
